test/api/ticket/samples.go

```diff
@@ -0,0 +1,22 @@
+package ticket
+
+import (
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+var Samples = []api.Ticket{
+	{
```

Comment:
I'm not sure about typical values of the Ticket, but values of most of fields below might need be updated. Need find some example values.